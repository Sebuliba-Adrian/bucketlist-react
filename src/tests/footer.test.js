import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/authentication/footer';

const props = {
  message: 'success',
  link: '/login',
  linkText: 'login',
};

it('renders correctly', () => {
  const footer = renderer.create(<Footer {...props} />).toJSON();
  expect(footer).toMatchSnapshot();
});
