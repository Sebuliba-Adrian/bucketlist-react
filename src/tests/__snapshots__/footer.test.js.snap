// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="card-text text-center grey-text message mt-3"
>
  success
  <a
    className="card-link"
    href="/login"
  >
    login
  </a>
</p>
`;
